app/models/grda_warehouse/tasks/service_history/enrollment.rb

```diff
@@ -762,12 +762,15 @@ def build_until
           exit_date,
           (Date.current + 1.year).end_of_year,
         ].min
-      else
+      elsif export.present?
         [
           export.effective_export_end_date,
           export.ExportEndDate,
           Date.current,
         ].compact.min.to_date
+      else
+        # If there is no Export, the data is coming from the OP HMIS. Treat it as accurate up to the current date.
+        [Date.current]
```

Comment:
I thought I'd fixed this on some branch.  You can just change line 767 and 768 to have `&` like
```
export&.effective_export_end_date,
export&.ExportEndDate,
```
Should have the same result.

Reply:
It's not on the release branch, which is what I'm working from

Reply:
Yeah, where ever I implemented it, it hasn't been merged.